.pep8speaks.yml

```diff
@@ -0,0 +1,14 @@
+# File : .pep8speaks.yml
+ 
+scanner:
+    diff_only: True  # If False, the entire file touched by the Pull Request is
+                     # scanned for errors. If True, only the diff is scanned.
+    linter: pycodestyle  # Other option is flake8
+ 
+pycodestyle:  # Same as scanner.linter value. Other option is flake8
+    max-line-length: 90  # Default is 79 in PEP 8
```

Comment:
Can we keep the default here?
In your example [here](https://github.com/PyAbel/PyAbel/issues/268#issuecomment-574984497) I can see only 88 characters:
![longline](https://user-images.githubusercontent.com/12854133/80153941-c1603200-857b-11ea-9afc-46b4b6071417.png)
We can always ignore this warning when needed, but using the standard value is probably better than some random number.

Reply:
Agreed. We should also tell people that it's okay to break the rules sometimes. I think that we can change the message from pep8speaks, so maybe we can change it to something that refers folks to our style policy: "try to follow PEP8 unless breaking the rules makes your code better."